.github/workflows/codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: '30 8 * * *'
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: ${{ 'ubuntu-latest' }}
+    permissions:
+      # required for all workflows
```

Comment:
Lets clear up the comments, this one is only required to publish the results? Not for `required for all workflows`

Reply:
Yeah, let me remove it, this comment is irrelevant. You are correct, it's only to publish results.